.github/workflows/pr-lint.yml

```diff
@@ -50,5 +50,15 @@ jobs:
           LABEL_NAMES: ${{ toJson(github.event.pull_request.labels.*.name) }}
         if: contains(env.LABEL_NAMES, 'under review') == contains(env.LABEL_NAMES, 'in progress')
```

Comment:
This needs to be updated.

Reply:
Forgot to remove

Reply:
as we have 3rd label to check, I decided to use `&&` https://github.com/KomodoPlatform/komodo-defi-framework/pull/2295/commits/3678a8bb4dd45a994a5da917d921b24ad67e94d2